src/types/k8s.go

```diff
@@ -40,6 +40,10 @@ type InstalledChart struct {
 	ChartName string `json:"chartName"`
 }
 
+type InstalledBigBang struct {
```

Comment:
?

Reply:
Can delete this.  I saw the "InstalledChart" object and thought I was going to have to keep track of what's installed in a particular way that never was realized.